src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/TermsOfUseAccessRequirementWidgetViewImpl.java

```diff
@@ -95,6 +100,16 @@ public void setTerms(String arText) {
 		terms.setHTML(arText);
 	}
 
+	@Override
+	public void setAccessRequirementID(String arID) {
+		accessRequirementIDField.setText(arID);
+	}
+
+	@Override
+	public void setAccessRequirementIDVisible() {
+		accessRequirementIDUI.setVisible(true);
+	}
```

Comment:
Could you just make default visibility true and remove this method?

Reply:
Actually maybe not, I see that a non ACT user may see this view and shouldn't see the ID